src/deploy-tes-on-azure/KubernetesManager.cs

```diff
@@ -567,18 +591,18 @@ void OutputHandler(object sendingProcess, DataReceivedEventArgs outLine)
             process.Start();
             process.BeginOutputReadLine();
             process.BeginErrorReadLine();
-            await process.WaitForExitAsync();
+            await process.WaitForExitAsync(cToken);
```

Comment:
Consider process.Kill() if cToken's cancelation state has been triggered.

Reply:
Internally WaitForExitAsync will kill it if the cTaken is triggered.

Reply:
I didn't see that in the sources for it